explorer-ui/src/components/ProfileView/Preferences/Preferences.jsx

```diff
@@ -75,10 +77,36 @@ function handleOccupationsSelected(event, occupation) {
     setPreferenceInfo({"interests": preferenceInfo.interests, "attractions": preferenceInfo.attractions, "occupation": preferenceInfo.employment, "budget": preferenceInfo.budget, "currLocation": currLocation, "visibility": preferenceInfo.visibility})
     console.log("current location is: ", currLocation)
   };
+
+
+// Send the profile information to the database 
+  function handleSubmit(event) {
+    console.log("submitted")
+    event.preventDefault();
+    const preferences = async () => {
+      try {
+        const res = await axios.post(
+          `${config.API_BASE_URL}/profileInfo`,
+          {
+            preferenceInfo: preferenceInfo
+          },
+          { maxContentLength: Infinity, maxBodyLength: Infinity }
```

Comment:
wild xD

Reply:
lol this was supposed to fix my issue with the base64 conversion files being too big but it didn't help